src/components/Item-form/ItemForm.jsx

```diff
@@ -0,0 +1,142 @@
+          >
+            <h2 className=" text-center font-bold uppercase">Add New Item</h2>
+            <div className="w-72 md:w-96">
+              <label htmlFor="name" className="mb-2 block">
+                <p className="font-semibold">Name</p>
+                <input
+                  type="text"
+                  id="name"
+                  value={name}
+                  onChange={(e) => setName(e.target.value)}
+                  placeholder="Marker Kit"
+                  className="my-2 w-full border border-[#BFD872] px-6 py-2 focus:border-[#85a80d] focus:bg-white focus:outline-none"
+                  required
+                />
+              </label>
+              <label htmlFor="description" className="mb-2 block">
+                <p className="font-semibold">Description</p>
+                <input
+                  type="text"
+                  id="description"
+                  value={description}
+                  placeholder="A nice marker kit"
+                  onChange={(e) => setDescription(e.target.value)}
+                  className="my-2 w-full border border-[#BFD872] px-6 py-2 focus:border-[#85a80d] focus:bg-white focus:outline-none"
+                  required
+                />
+              </label>
+              <label htmlFor="image" className="mb-2 block">
+                <p className="font-semibold">Image</p>
+                <input
+                  type="text"
+                  id="image"
+                  value={image}
+                  placeholder="https://vespaindia.com/images/classic/vxl_yellow.webp"
+                  onChange={(e) => setImage(e.target.value)}
+                  className="my-2 w-full border border-[#BFD872] px-6 py-2 focus:border-[#85a80d] focus:bg-white focus:outline-none"
+                  required
+                />
+              </label>
+              <label htmlFor="price" className="mb-2 block">
+                <p className="font-semibold">Price</p>
+                <input
+                  type="text"
+                  id="price"
+                  value={price}
+                  placeholder="Price"
+                  onChange={(e) => setPrice(e.target.value)}
+                  className="my-2 w-full border border-[#BFD872] px-6 py-2 focus:border-[#85a80d] focus:bg-white focus:outline-none"
+                  required
+                />
+              </label>
+              <button
+                type="submit"
+                className="mt-4 bg-[#97BF0F] px-6 py-2 text-white hover:bg-[#85a80d] focus:bg-[#6d8a0a]"
+              >
+                Submit
+              </button>
+              <button
+                type="button"
+                className="ml-5 mt-4 bg-[#E4E5E9] px-6 py-2 text-black hover:bg-[#cacacc] focus:bg-[#9a9a9c]"
+                onClick={() => redirect(-1)}
+              >
+                Cancel
+              </button>
+            </div>
+          </form>
```

Comment:
Some of these elements are reptitive. Don't forget DRY.

Consider using a subcomponent for form inputs like so:

```js
const FormInput = ({ id, value, placeholder, onChange, className, labelText }) => (
  <label htmlFor={id} className="mb-2 block">
    <p className="font-semibold">{labelText}</p>
    <input
      type="text"
      id={id}
      value={value}
      placeholder={placeholder}
      onChange={onChange}
      className={className}
      required
    />
  </label>
);

FormInput.propTypes = {
  id: PropTypes.string.isRequired,
  value: PropTypes.string.isRequired,
  placeholder: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired,
  className: PropTypes.string.isRequired,
  labelText: PropTypes.string.isRequired,
};
```